SIS_Images_Scraper.py

```diff
@@ -0,0 +1,204 @@
+
+    # loop through list of students to get image, name, and email
+    # all info collected from for loop (img url, name, email) put into dict
+    for s in range(1, len(student_list)):
+        student_record = {}
+        student = driver.find_elements_by_class_name('datadisplaytable')[2].find_element_by_tag_name('tbody').find_elements_by_tag_name('tr')[s]
+        student.find_elements_by_tag_name('td')[1].find_element_by_class_name('fieldmediumtext').click()
+
+        img_url = driver.current_url
+        driver.get(img_url)
+
+        # image
+        image = driver.find_elements_by_tag_name('img')[6].get_attribute('src')
+        student_record['img url'] = image
+
+        # name
+        info_name = driver.find_elements_by_class_name('plaintable')[4].find_element_by_tag_name('tbody').find_element_by_tag_name('tr').find_elements_by_tag_name('td')[1].text
+        name = info_name[16:]
+        student_record['name'] = name
+
+        # email address
+        driver.find_element_by_link_text('Student E-mail Address').click()
+        if len(driver.find_elements_by_class_name('datadisplaytable')) == 1:
+            emails = driver.find_element_by_class_name('datadisplaytable').find_element_by_tag_name('tbody').find_elements_by_tag_name('tr')
+            for i in range(len(emails)):
+                if emails[i].text == "Campus Student Email Address":
+                    email = emails[i+1].find_element_by_tag_name('td').text
+                    student_record['email'] = email
+                    break
+        class_list.append(student_record)
+        driver.back()
+        driver.back()
+    driver.back()
+    driver.back()
+    return class_list
+
+
+# Gets the info regarding each course of student images with their rcs id
+def getInfoFromCourse(driver):
+    # Get the term to use to save images
+    term = getSession(driver)
+    if term == "Exit":
+        return
+
+    # click Course Information- Select a CRN
+    driver.find_element_by_link_text('Course Information- Select a CRN').click()
+
+    # iterate and ask if user wants images/names from this course
+    select_course = Select(driver.find_element_by_name('crn'))
+    options_course = select_course.options
+    for index in range(len(options_course)):
+        # all dicts put into list for each class section
+        class_list = []
+        select_course = Select(driver.find_element_by_name('crn'))
+        options_course = select_course.options
+        course = options_course[index].text
+
+        # gets the images the user wants for the class section by looping until the user enters a valid command
+        foundAnswer = False
+        while not foundAnswer:
+            # Asks if user wants pictures from current course displayed
+            print("Do you want pictures from {}?".format(course))
+            answer = input("Y/N/Exit\n")
```

Comment:
`answer = input("Y/N/Exit\n").lower()` unless you're explicitly doing something with a chance of breaking someone's computer, please lowercase (or uppercase) an answer and then do the comparison.